pkg/skaffold/schema/validation/validation.go

```diff
@@ -187,3 +188,18 @@ func validatePortForwardResources(pfrs []*latest.PortForwardResource) []error {
 	}
 	return errs
 }
+
+// validateJibPluginTypes makes sure that jib type is one of `maven`, or `gradle` if set.
+func validateJibPluginTypes(artifacts []*latest.Artifact) (errs []error) {
+	for _, a := range artifacts {
+		if a.JibArtifact == nil || a.JibArtifact.Type == "" {
+			continue
+		}
+		t := strings.ToLower(a.JibArtifact.Type)
+		if t == "maven" || t == "gradle" {
+			continue
+		}
+		errs = append(errs, fmt.Errorf("artifact %s has invalid plugin type '%s'", a.ImageName, t))
```

Comment:
Is it possible that `a.ImageName` is empty?

"... has invalid Jib plugin type..." ?

Reply:
Image names are validated further above in `validateImageNames()`.  There's not much we can do at this point.